src/dropbox.js

```diff
@@ -66,6 +66,19 @@ export default class Dropbox {
   }
 
   request(path, args, auth, host, style) {
+    // checks for multiauth and assigns auth based on priority to create header in switch case
+    if (auth.split(',').length > 1) {
+      const authTypes = auth.replace(' ', '').split(',');
+      const multiAuth = new Set(authTypes);
```

Comment:
Small nit: you don't actually need this line, you can iterate through the authTypes object.